docs/reference/configuration/observations.rst

```diff
@@ -72,29 +72,29 @@ created as follows:
 This will create an observation of group oil production for the Brent
 group on 21th of august 2005. The observed value was 100 with a
 standard deviation of 5. The name SEP_TEST_2005 will be used as a
-label for the observation within the ERT and must be unique.
+label for the observation within ERT and must be unique.
 
 Date format YYYY-MM-DD (ISO 8601) is required.
 Other time formats, like DD/MM/YYYY or DD.MM.YYYY, are deprecated
-and its support will be removed in a future release. The date format
+and their support will be removed in a future release. The date format
 can also include hours and seconds: "YYYY-MM-DDTHH:mm:ss". When the
 summary file is read from the realization, report times are rounded
 to seconds and matched to closest observations with 1 second tolerance.
 
-
 The item KEY in a SUMMARY_OBSERVATION is used to look up the simulated value
-from the summary file. To condition on the summary key VAR in well, group or
-region WGRNAME, one uses::
+from the summary file. To condition on the summary key VAR in a well, group or
+region WGRNAME, use::
 
  KEY = VAR:WGRNAME;
 
-For example, to condition on RPPW in region 8, one uses::
+For example, to condition on RPPW in region 8, use::
 
  KEY = RPPW:8;
 
-It is also possible to give the observation time as a restart number
-using the RESTART item or as time in days from simulation start using
-the DAYS item. Use a floating point DAYS item for clock-times.
+Use the keyword ``RESTART`` to specify observation time as a restart number.
+Use the keyword ``DAYS`` to specify observation time as the number of days relative
+to ``START`` as defined in the Eclipse DATA file.
```

Comment:
This is not always the case, if you use `TIMEMAP` instead of `REFCASE` it points to the first index in the time map. Also people could be using another simulator with a similar format. Suggest rephrasing to:
```
Use the keyword ``DAYS`` to specify observation time as the number of days relative
to the start of the simulaton, where the start point is taken from the `REFCASE` or `TIMEMAP`
```